### PR TITLE
adding support for SNS topic and multi region

### PR DESCRIPTION
**Pull request by ricoli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,36 @@ The following AWS Config Rules are supported:
 
 ## Usage
 
-    module "aws_config" {
-      source             = "trussworks/config/aws"
-      config_logs_bucket = "my-aws-logs"
-    }
+```terraform
```
Comment (dynamike) [inline README.md]: This is populated by `terraform-docs`  in https://github.com/trussworks/terraform-aws-config/blob/master/main.tf#L1

File: variables.tf [hunk 2]
```diff
@@ -70,3 +70,8 @@ variable "check_rds_public_access" {
   description = "Enable rds-instance-public-access-check rule"
   default     = false
 }
+
+variable "sns_topic_arn" {
+  description = "Optional. An SNS topic to stream configuration changes and notifications to."
```
Comment (dynamike) [inline variables.tf]: No need to include Optional. terraform-docs handles this by generating https://github.com/trussworks/terraform-aws-config#inputs

**Comment by dynamike** [comment]
Looks like you've also got merge conflicts.

**Comment by ricoli** [comment]
> Looks like you've also got merge conflicts.

Sorted :)

File: README.md (hunk 1 above)
Comment (dynamike) [reply README.md]: Sorry, I don't think I was clear in the original review. The README is generated using a pre-commit hook, that runs something called `terraform-docs`. You'll want to move this usage update to https://github.com/trussworks/terraform-aws-config/blob/master/main.tf#L1  and run the pre-commit hooks. You'll need to install [pre-commit ](https://pre-commit.com/#install) if you don't already have it.

```
pre-commit install --install-hooks
pre-commit run --all-files
```

Comment (ricoli) [reply README.md]: I see what you mean - you may want to add a CONTRIBUTING file to this repo in the future to explaining this sort of stuff :)

Comment (ricoli) [reply README.md]: should be sorted now

**Comment by gibbsoft** [comment]
Would be very useful to get this merged.

**Comment by dynamike** [comment]
At this point the module has been modified to be able to support multiple regions. Could you resubmit a PR on the SNS topic changes?

**Pull request closed without merging**